functions.go

```diff
@@ -1,114 +1,118 @@
 package sqlparser
 
-var AllowedFunctions = map[string]struct{}{
+var AllowedFunctions = map[string]bool{
```

Comment:
I've added a flag to the `AllowedFunctions` map to indicate if a function is custom and included the two custom functions to the map: `txn_hash` and `block_num`